torchgeo/transforms/indices.py

```diff
@@ -0,0 +1,234 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""TorchGeo indices transforms."""
```

Comment:
It might be useful to add links to those resources you used for index definitions and citations so that we don't forget about them.

Reply:
Added links, let me know if it works for you.